core/src/main/res/values/strings.xml

```diff
@@ -244,4 +244,20 @@
     <string name="core_match_device">Match device</string>
     <string name="core_match_device_description">Matches your device settings automatically.</string>
 
+    <!-- Certificate Preview -->
+    <string name="iap_certificate_verified_text">Verified</string>
+    <string name="iap_certificate_text">Certificate</string>
+    <string name="iap_certificate_certify_message_1">This is to certify that</string>
+    <string name="iap_certificate_certify_message_2">has successfully completed all courses and received passing grades for a Verified Certificate in</string>
+    <string name="iap_certificate_organization_message">a course offered by %1$s, an online learning partnership between %1$s and %2$s.</string>
+    <string name="iap_certificate_verified_certificate_text">Verified Certificate</string>
+    <string name="iap_certificate_issued_date_text">Issued April 2016</string>
```

Comment:
IMO, better to use the current date or a future date for dummy data to make it more relevant